src/main/java/com/epam/esm/controller/GiftCertificateController.java

```diff
@@ -67,19 +67,10 @@ public void updateCertificate(@RequestBody Certificate certificate) throws DaoEx
         }
     }
 
-    @RequestMapping(path = "/getCertificateByName", method = RequestMethod.GET)
-    public List<Certificate> searchCertificateByName(@RequestParam String name) throws DaoException {
+    @RequestMapping(path = "/search", method = RequestMethod.GET)
+    public List<Certificate> search (@RequestParam String template) throws DaoException {
```

Comment:
we can remove this method and use instead method getAll with two optional parameters: name and description.

Reply:
Removed this method, now i can search with two optional parameters